starters/gatsby-wordpress-starter/src/templates/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+import { Link, graphql } from "gatsby"
+
+import Layout from "../components/layout"
+import Seo from "../components/seo"
+import { withGrid, PostGridItem } from "../components/grid"
+
+const PageHeader = () => (
+  <div className="prose sm:prose-xl mt-20 flex flex-col mx-auto max-w-fit">
+    <h1 className="prose text-4xl text-center h-full">
+      Welcome to{" "}
+      <a
+        className="text-purple-600 no-underline hover:underline"
+        href="https://www.gatsbyjs.com/"
+      >
+        Gatsby!
+      </a>
+    </h1>
+
+    <div className="text-2xl">
+      <div className="not-prose bg-black text-white rounded flex items-center justify-center p-4">
+        Decoupled WordPress on{" "}
+        <img src="/pantheon.png" alt="Pantheon Logo" width={191} height={60} />
+      </div>
+    </div>
+  </div>
+)
+
+const Index = ({ data, pageContext: { nextPagePath, previousPagePath } }) => {
+  const posts = data.allWpPost.nodes
+
+  const PostGrid = withGrid(PostGridItem)
+
+  return (
+    <Layout isHomePage>
+      <Seo title="All posts" />
+      <PageHeader />
+
+      <section>
+        <PostGrid data={posts} />
+      </section>
+      {previousPagePath && (
+        <>
+          <Link to={previousPagePath}>Previous page</Link>
+          <br />
+        </>
+      )}
+      {nextPagePath && <Link to={nextPagePath}>Next page</Link>}
```

Comment:
The link seems a bit disconnected from the content. Can you add some margin and padding and maybe add an underline and arrow like we do on a Post (see below)?

![image](https://user-images.githubusercontent.com/63015754/186532206-be0c2deb-218f-4c59-bfdb-8db1a987911f.png)

Reply:
I also noticed the homepage has 2 pages with the second page consisting of 2 posts, but the `/posts` route has all of the posts on 1 page.